spec/unit/type/rpmkey_spec.rb

```diff
@@ -71,5 +71,16 @@
       end
     end
 
+    describe "autorequire" do
+      it "should autorequire a local file" do
+        catalog = Puppet::Resource::Catalog.new
+        file = Puppet::Type.type(:file).new(:name => '/tmp/foo', :content => 'bar' )
+        catalog.add_resource file
+        key = described_class.new(:name => 'DB42A60E', :source => '/tmp/foo', :ensure => :present)
+        catalog.add_resource key
+        key.autorequire.size.should == 1
+      end
+    end
```

Comment:
2) yes,
1) I wondered about that, but I didn't come up with a test case that actually makes sense. I mean do we really need to test puppet's autorequire functionality?